src/main/proto/TokenMint.proto

```diff
@@ -33,7 +33,16 @@ The operation increases the Total Supply of the Token. The maximum total supply
 The amount provided must be in the lowest denomination possible. Example:
 Token A has 2 decimals. In order to mint 100 tokens, one must provide amount of 10000. In order to mint 100.55 tokens, one must provide amount of 10055.
  */
+
+message AmountOrMemo {
```

Comment:
Do we need this to be a separate message?

Reply:
Even if `deprecated`, the amount is still in the `TokenMintTransactionBody`, so I created the `AmountOrMemo` according to the HIP. I was looking for a way to maintain naming. We can put this directly into `TokenMintTransactionBody` and change some property names, but that's up to you.